package.json

```diff
@@ -43,29 +43,35 @@
 		"keychain": "^1.4.0",
 		"keytar": "^7.9.0",
 		"lodash": "^4.17.13",
-		"mkdirp": "^1.0.4",
+		"mkdirp": "^3.0.1",
 		"nanoid": "^3.3.1",
 		"open": "^8.0.4",
 		"ora": "^5.4.1",
 		"p-event": "^2.3.1"
 	},
 	"devDependencies": {
-		"@oclif/dev-cli": "^1.19.5",
-		"@oclif/test": "^1.1.0",
+		"@oclif/prettier-config": "^0.2.1",
+		"@oclif/test": "^3.2.15",
+		"@typescript-eslint/eslint-plugin": "^8.17.0",
 		"chai": "^4.1.2",
 		"chai-as-promised": "^7.1.1",
-		"eslint": "^5.3.0",
-		"eslint-plugin-node": "^7.0.1",
-		"eslint-plugin-promise": "^3.8.0",
+		"eslint": "^8.57.1",
+		"eslint-config-oclif": "^5",
+		"eslint-config-oclif-typescript": "^3",
+		"eslint-config-prettier": "^9",
+		"eslint-plugin-n": "^17.14.0",
+		"eslint-plugin-promise": "^7.2.1",
 		"eslint-plugin-unicorn": "^5.0.0",
 		"jsonwebtoken": "^8.3.0",
-		"leche": "^2.2.3",
-		"mocha": "^10.0.0",
+		"leche": "^2.3.0",
+		"mocha": "^10.8.2",
 		"mockery": "^2.1.0",
-		"nock": "^10.0.0",
-		"nyc": "^15.1.0",
-		"sinon": "^15.0.1",
-		"standard-version": "^9.5.0"
+		"nock": "^13.5.6",
+		"nyc": "^17.1.0",
+		"oclif": "^4.16.0",
+		"sinon": "^19.0.2",
+		"standard-version": "^9.5.0",
+		"typescript": "^5.7.2"
```

Comment:
Is it required for oclif now?

Reply:
If without this package, we will have a warning about we should install oclif.

Reply:
I think it's a fine if it's a dev dependency